doc/environment.yml

```diff
@@ -8,8 +8,6 @@ dependencies:
   - pandas
   - matplotlib
   - pytables
-  - jupyter
-  - jupyter_client
   - sphinx
   - numpydoc
   - nbsphinx
```

Comment:
the commit message tells about nbsphinx not being required either???

Reply:
nbsphinx is an extension of sphinx (see conf.py). I should change the commit message.

Reply:
what about
"updated doc/conf.py (remove nbsphinx extension) and doc/environment.yml (removed jupyter and jupyter_client packages): jupyter scripts no longer used in documentation" for the commit message?

Reply:
Whatever. If it's only the commit message wrong, no need to change it.